api/src/main/java/jakarta/persistence/spi/TransformerException.java

```diff
@@ -11,11 +11,13 @@
  */
 package jakarta.persistence.spi;
 
+import jakarta.persistence.PersistenceException;
+
 /**
  *
  * @since 3.1
  */
-public class TransformerException extends Exception {
+public class TransformerException extends PersistenceException {
```

Comment:
I'm wondering if PersistenceException is the right Exception for the job here.  If a failed transform causes a PersistenceException to be thrown, will it mark the transaction for rollback?  I'm wondering if maybe making the parent of TransformException to be RuntimeException instead might be better.

Reply:
(saw the PersistenceExecption javadoc update after re-review).  So whether to use RuntimeException instead is my only outstanding question.

Reply:
Thinking about it, RuntimeException might not be the best either (burden of dealing with unchecked exceptions with the transformer caller).  But I am rather iffy on using PersistenceException, since its general use is from EntityManager operations, queries, or interactions with the persistence context.  I can't really think of any time PersistenceException is thrown outside of those scenarios.

Reply:
Ok, updated TransformerException to regular (not runtime) ex